docs/source/conf.py

```diff
@@ -78,9 +79,15 @@
 
 autoclass_content = "both"
 autodoc_typehints = "both"
+autodoc_typehints_format = "short"
 autosummary_generate = False
 autodoc_member_order = "groupwise"
 autodoc_docstring_signature = True
+python_use_unqualified_type_names = True
```

Comment:
just a suggestion: we didn't write a comment on any of those sphinx settings and it's starting to be a lot of them, not always clear on a why. What do you think of adding a short comment on a why it is here?

Reply:
Good idea, I'll add comments to explain their roles